### PR TITLE
feat(pagination): update layout to support pagination widget

### PR DESCRIPTION
**Pull request by gabaldon**

### CODE REVIEW

File: lib/util/storage/database/wallet.dart [hunk 1]
```diff
@@ -117,7 +122,30 @@ class Wallet {
         if (vtt.status != 'unknown hash') _vttMap[vtt.txnHash] = vtt;
       });
     });
-    return _vttMap.values.toList();
+    return _vttMap.values.toList()
+      ..sort((t1, t2) => t2.txnTime.compareTo(t1.txnTime));
+  }
+
+  PaginatedData paginatedTransactions(PaginatedDataArgs args) {
```
Comment (Tommytrg) [inline lib/util/storage/database/wallet.dart:129]:
```suggestion
  PaginatedData getPaginatedTransactions(PaginatedDataArgs args) {
```

File: lib/widgets/layouts/dashboard_layout.dart [hunk 2]
```diff
@@ -34,10 +34,12 @@ class DashboardLayout extends StatefulWidget {
   final ScrollController? scrollController;
   final Widget dashboardChild;
   final List<Widget> actions;
+  final Function(PaginatedDataArgs)? getPaginatedData;
```
Comment (Tommytrg) [inline lib/widgets/layouts/dashboard_layout.dart:37]:
```suggestion
  final Function(PaginatedDataArgs)? getPaginatedTransactions;
```

**Comment by Tommytrg** [comment]
The main blocker to merge this PR is a problem with the scroll. The behaviour of the touchpad and the mouse wheel is different: it triggers the events needed for the pagination but not with the mouse.

**Comment by Tommytrg** [comment]
Could we close this issue now that we have merged  #234 ?

**Comment by gabaldon** [comment]
Decided to add number paginator instead #234

**Pull request closed without merging**